app/services/preserver/blind_importer.rb

```diff
@@ -33,6 +33,7 @@ def import!
     end
     # Get rid of non-preserved members.
     source_change_set.try(:member_ids=, member_ids)
+    source_change_set.resource.new_record = true
```

Comment:
?

Reply:
https://github.com/samvera/valkyrie/pull/867/commits/d81d2ae2562a86a5767ed945ecfa59ffe3ebe918#diff-a006f149ad43c693a7064a57d903dacc5dcc4a4609d21ed2f54340c9fbd3829eL34-L35

The following is a little fuzzy:

Valkyrie used to have a hack in it to pretend records were new so it wouldn't perform ID or optimistic locking checks. Now there's external_resource: true in CompositePersister so it wasn't necessary. Unfortunately we were relying on that for importers, which create new records by duplicating the previous object.

It might be we can use external_record: true here, but I don't think ChangeSetPersister provides access to that quite yet.

Reply:
I think I can get rid of this, running tests locally.